### PR TITLE
feat: PUT on calendar and addressbook objects adds response headers L…

### PR DESCRIPTION
**Pull request by DeepDiver1975**
…ocation, ETag and Last-Modified

Finding while running ccs-caldavtester on OCIS ....

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@emersion @bitfehler please review - THX a lot

**Comment by DeepDiver1975** [comment]
@emersion how can we get this moving? THX

**Comment by bitfehler** [comment]
@DeepDiver1975 I was out on vacation. I'll try to get to it this week.

**Comment by bitfehler** [comment]
LGTM, thanks a lot. Just the same general nitpicks as for #140 - could you make the commit message have a shorter summary, s/t like: "carddav/caldav: add missing headers on PUT" and then elaborate in the description?

File: server.go [hunk 1]
```diff
@@ -193,18 +193,24 @@ func (b *backend) PropPatch(r *http.Request, update *internal.PropertyUpdate) (*
...
+		return err
+	}
+	err = wc.Close()
+	if err != nil {
+		return err
 	}
 
-	return nil, wc.Close()
+	w.WriteHeader(http.StatusCreated)
+
```
Comment (emersion) [inline server.go:212]: Carried over the TODOs here as well

**Comment by emersion** [comment]
Edited the commit message.

Thanks!

**Pull request merged**